src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -57,13 +57,11 @@ export class TaskListBuilder {
   static buildRespondToClaimSection (draft: ResponseDraft, claim: Claim): TaskList {
     const externalId: string = claim.externalId
     const tasks: TaskListItem[] = []
-
     tasks.push(
       new TaskListItem(
         'Choose a response',
         Paths.responseTypePage.evaluateUri({ externalId: externalId }),
-        OweMoneyTask.isCompleted(draft)
-      )
+        OweMoneyTask.isCompleted(draft))
```

Comment:
You didnt make any change in this file, PR will be cleaner if you just revert this file